src/bootstrap/src/core/builder.rs

```diff
@@ -439,7 +439,9 @@ impl StepDescription {
         }
 
         // strip CurDir prefix if present
-        let mut paths: Vec<_> = paths.iter().map(|p| p.strip_prefix(".").unwrap_or(p)).collect();
+        let curdir_path = PathBuf::from(".").canonicalize().unwrap();
```

Comment:
Canonicalize can fail on Windows.

Reply:
I realized I don't need to canonicalize this path anymore, because I can get an equivalent absolute path from `builder.src`.

However, I am now using `.canonicalize()` elsewhere. If you think this will be a problem, please leave another comment.